inference-engine/tests/functional/plugin/shared/include/behavior/core_threading_tests.hpp

```diff
@@ -140,6 +143,8 @@ TEST_P(CoreThreadingTests, smoke_GetConfig) {
 
 // tested function: GetMetric, UnregisterPlugin
 TEST_P(CoreThreadingTests, smoke_GetMetric) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
```

Comment:
why do we need for `CoreThreadingTests`? Looks like problems only with `CoreThreadingTestsWithIterations`

Reply:
Yes, sure. But I made changes everywhere just in case